randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1142,23 +1142,36 @@ Extra - room_id: [28]
   * Extra - blockx: 9
   * Extra - blocky: 6
   > Other to Reservoir East
-      Morph Ball
+      Any of the following:
+          After Sector 4 Water Level Lowered
+          Damage Runs (Beginner) and Electrified Water Damage ≥ 100
```

Comment:
not critical since you need morph to get it, but probably safe to have this all in an `AND morph`